docs/learn/create-a-blog/use-blog-post-data-in-template.md

```diff
@@ -41,6 +41,7 @@ export interface ScullyRoute {
   published?: boolean;
   slug?: string;
   sourceFile?: string;
+  lang?: string;
```

Comment:
this is a convention we use in our docs, but it is not something we should put in the samples?